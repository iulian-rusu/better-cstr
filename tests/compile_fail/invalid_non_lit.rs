use better_cstr::c;

fn main() {
    c!(std::ops::Add::add(34, 35));
}
