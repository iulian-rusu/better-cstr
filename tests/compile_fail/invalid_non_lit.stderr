error: expected literal
 --> tests/compile_fail/invalid_non_lit.rs:4:8
  |
4 |     c!(std::ops::Add::add(34, 35));
  |        ^^^
